package.json

```diff
@@ -1,55 +1,55 @@
 {
-  "author": "Jared Allard <jaredallard@outlook.com>",
   "name": "nexe",
-  "description": "create single executables out of your [node/io].js applications",
+  "description": "Create a single executable out of your Node.js application",
   "license": "MIT",
-  "version": "1.1.2",
+  "version": "2.0.0-beta.1",
   "contributors": [
-    {
-      "name": "Criag Condon",
-      "email": "craig.j.condon@gmail.com",
-      "url": "http://crcn.io/"
-    }
+    "Craig Condon <craig.j.condon@gmail.com> (http://crcn.io)",
+    "Jared Allard <jaredallard@outlook.com>",
+    "Caleb Boyd <caleb.boyd@hotmail.com>"
   ],
   "scripts": {
-    "test": "echo node: $(node -v) && node_modules/.bin/mocha --reporter mocha-circleci-reporter test/test.js"
+    "clean": "rimraf lib",
```

Comment:
I'm not sure what you mean. When you're in an npm run script the contents of node_modules/.bin is added to the path so you can use local modules in the scripts. (eg. that statement doesn't require global rimraf)